module/programs/helper.js

```
@@ -1579,4 +1579,37 @@ module.exports = class ProgramsHelper {
+          success: true,
+          message: messageConstants.apiResponses.PROGRAM_IMPORTED,
+          result: programDocument,
+        });
+      } catch (error) {
+        return resolve({
+          success: false,
+          status: error.status ? error.status : httpStatusCode['internal_server_error'].status,
+          message: error.message,
+        });
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)